lib/ae_mdw/sync/server.ex

```diff
@@ -125,4 +152,17 @@ defmodule AeMdw.Sync.Server do
   end
 
   defp height, do: :aec_headers.height(:aec_chain.top_header())
+
+  defp commit_mutations(state, {_height, mbi}, block, mutations) do
+    new_state = State.commit(state, mutations)
+
+    Producer.commit_enqueued()
+
+    if mbi != -1 do
+      Broadcaster.broadcast_micro_block(block, :mdw)
+      Broadcaster.broadcast_txs(block, :mdw)
+    end
+
+    new_state
+  end
```

Comment:
After key block commit it would be a time also for `Broadcaster.broadcast_key_block(key_block, :mdw)`

Reply:
Added it, thanks!